storage/object/reference-content/cname-record.mdx

```diff
@@ -0,0 +1,84 @@
+
+| Subdomain                                  | Target host                                    |
+|--------------------------------------------|------------------------------------------------|
+| `my-chosen-subdomain`                      | `bucket-name.edge.scw.cloud`                   |
+
+| Host record                                | Points to                                      |
+|--------------------------------------------|------------------------------------------------|
+| `my-chosen-subdomain`                      | `bucket-name.edge.scw.cloud`                   |
+
+| `my-chosen-subdomain`                      | Record Type     | Value                                          |
+|--------------------------------------------|-----------------|------------------------------------------------|
+| `@`                                        | `CNAME`         | is an alias of `bucket-name.edge.scw.cloud`    |
+
+
+You may also see a `TTL`field, which stands for **T**ime **T**o **L**ive.  This tells the DNS resolver how long it can cache this record, before it must re-check the original source in case something has changed. TTL is measured in seconds, and the default value is usually 12 hours (43200 seconds) or 24 hours (86400 seconds). 
+
+ ## Troubleshooting DNS errors
+
+When setting up your customized subdomain with Edge Services, you have the option to carry out a verification check on the CNAME record. Edge Services will query the subdomain and check that it resolves correctly to the Edge Services endpoint. If there is a problem, you will see an error message. 
+
+    TODO SCREENSHOT
+
+An error message may also display at a later point from your Edge Services dashboard if a problem is detected at any point with your CNAME record or subdomain.
+
+See the table below for help with troubleshooting these errors:
+
+| Error message                             | Solution                                                            |
+|-------------------------------------------|---------------------------------------------------------------------|
+| No CNAME record found                     | Make sure you have created a valid CNAME record where your subdomain points to the Edge Services endpoint.  |
+| Incorrect CNAME                           | Make sure your CNAME record points to the Edge Services endpoint in the format `bucket-name.edge.scw.cloud`, and that you have replaced `bucket-name` with the name of your bucket.                                            |
+| Domain does not exist                     | You must own the domain name you are attempting to configure. If you do not already own the domain name, you cannot create a CNAME record for it. Register the domain name, for example using our [Domains and DNS](/network/domains-and-dns/how-to/register-internal-domain/) product, then create a CNAME record for it.                                           |
```

Comment:
@RoRoJ, nice idea to suggest using Scaleway DNS.
In this case I believe the issue is that we try to resolve the domain and don't get an answer because nobody registered the domain (no authoritative DNS server was found), or maybe the user made a typo when entering the domain name in the Console.